__tests__/api.test.js

```diff
@@ -229,3 +229,17 @@ describe("POST to /api/articles/:article_id/comments", () => {
     expect(body.message).toBe("Bad Request");
   });
 });
+
+describe("GET api/users", () => {
+  it("should return a status code of 200 and return an array of user objects", async () => {
+    const { body } = await request(app).get("/api/users").expect(200);
+    expect(body.users.length).toBeGreaterThan(0);
```

Comment:
We could check the specific length because we are working with test data we do not need to worry about future changes 🙂